src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -262,7 +262,7 @@ void VehicleIMU::Run()
 
 			// rotate sensor clip counts into vehicle body frame
 			const Vector3f clipping{_accel_calibration.getBoardRotation() *
-				Vector3f{(float)accel.clip_counter[0], (float)accel.clip_counter[1], (float)accel.clip_counter[2]}};
+						Vector3f{(float)accel.clip_counter[0], (float)accel.clip_counter[1], (float)accel.clip_counter[2]}};
```

Comment:
@xdwgood it still complains, because of this change here. It's a `make format` misconfiguration, I actually also have it on my setup. So just revert the changes in here, and see that your PR only touches tailsitter.cpp.

Reply:
I tried to restore, but when I commit it, it showed that the check failed... Can you help me submit a change? Thank you